### PR TITLE
Accurately sorts the tasks by days in the queue

### LINKED ISSUE #408: Manager Dashboard:  Reports:  Ability to view appeals without a decision document
As a BVA Dispatch Manager, I would like to view a listing of all appeals the have not been able to be associated with a decision document so that I can manage and escalate the issue to a resolution as necessary.

## Acceptance Criteria
1. Verify that when navigated to the Missing Decision report page, a list of appeals missing a decision is displayed.
1. Verify that only users with "System Admin" and "Manage Claim Establishment" are authorized to access this page. 
1. Verify that all appeals that are ready for Claims Establishment but missing a decision document are displayed in this list.
1. Verify that for each appeal in the list, the following columns of information is displayed:
   - **Veteran** column is filled in with the VBMS ID of the appeal
   - **Decision Type** - the VACOLS Decision Type value for the claim
   - **Decision Date** - is filled in with the decision date retrieved from VACOLS. _Format:_  mm/dd/yyyy
   - **Days in Queue** - is filled in with the # of days the system tried to obtain the decision document VBMS
1. Verify that the list is ordered by _"Days in Queue"_ with the most days being at the top.
1. Verify that the Header contains the current date in the following format:  

     > Claims Missing Decision - [Today's Date - mm/dd/yyyy]
1. Verify that the standard application header is displayed with the following elements (Refer to mockup): 
    - **Application Name** - "Caseflow Dispatach" 
    - **Application Logo** - [logo file]
    - User Name menu 
1. Verify that the standard application Footer is displayed with **_Digital Service_** and **_Send Feedback_** links. 
## Mockup
screen shot 2017-01-12 at 4 26 37 pm

# Technical Notes
- Use a US Web Design Standards table and the Caseflow Dispatch layout.
- URL is /dispatch/missing-decision

# Related Stories
Decision Review: Show Decision PDF and information #405
Manager Dashboard: Reports: Ability to limit and show more Missing Document Claims #632
Manager Dashboard: Header: User Menu: Ability to logout #633

### PR DESCRIPTION
**Pull request by sharonwarner**
This PR updates the missing appeals page to use the created_at time stamp to count the number of days a task has been in the queue. 

Fixes #408 

- [ ] Unit tests pass
- [ ] Confirmed tasks appear in the correct order on the manager missing decision docs page
- [ ] Confirmed tasks appear as expected on other manager and worker views

### CODE REVIEW

File: app/models/task.rb [hunk 1]
```diff
@@ -154,6 +154,10 @@ def assigned_existing_ep?
     completion_status == self.class.completion_status_code(:assigned_existing_ep)
   end
 
+  def days_since_creation
```
Comment (shanear) [inline app/models/task.rb]: can we have a unit test for this?

File: app/controllers/establish_claims_controller.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@ def assign_existing_end_product
   end
 
   def unprepared_tasks
-    @unprepared_tasks ||= EstablishClaim.unprepared
+    @unprepared_tasks ||= EstablishClaim.unprepared.oldest_first
```
Comment (shanear) [inline app/controllers/establish_claims_controller.rb]: can we also add a test for this too?

File: app/models/task.rb (hunk 1 above)
Comment (sharonwarner) [inline app/models/task.rb]: done!

File: app/controllers/establish_claims_controller.rb (hunk 2 above)
Comment (sharonwarner) [inline app/controllers/establish_claims_controller.rb]: done!

**Pull request merged**